README.md

```diff
@@ -112,7 +65,38 @@ docker run -d -p 443:443 -v chainweb-db:target=/root/.local/share/chainweb-node/
 Further details can be found in the [README of the docker
 repository](https://hub.docker.com/r/kadena/chainweb-node).
 
-### Building from Source
+### Docker (bare metal)
+
+A docker image with just a bare chainweb-node binary and its dependencies is
+available at `ghcr.io/kadena-io/chainweb-node/ubuntu:latest`. It is up to the
+user to setup and manage the database and configure the node to their needs.
+
+```sh
+docker run -p 1789:1789 -p 80:80 --entrypoint=/chainweb/chainweb-node ghcr.io/kadena-io/chainweb-node/ubuntu:latest --help
+docker run -p 1789:1789 -p 80:80 --entrypoint=/chainweb/chainweb-node ghcr.io/kadena-io/chainweb-node/ubuntu:latest --print-config
+```
+
+Examples for running docker compose setups for chainweb-node for different usage scenarios
+can be found in [this repository](https://github.com/kadena-io/docker-compose-chainweb-node).
+
+### Ubuntu Linux
+
+The following packages must be installed the on the host system:
+
+```bash
+sudo apt-get update
+apt-get install ca-certificates libgmp10 libssl1.1 libsnappy1v5 libtbb2 zlib1g liblz4-1 libbz2-1.0 libgflags2.2 zstd
```

Comment:
For ubuntu-22.04 `libtbb2` is replaces by `libtbb12`. I wonder if we just ask for `libtbb-dev` which would work on both.